kubernetes/adapter.go

```diff
@@ -332,19 +338,23 @@ func (a *Adapter) ListIngress() ([]*Ingress, error) {
 	}
 	var ret []*Ingress
 	if len(a.ingressFilters) > 0 {
-		ret = make([]*Ingress, 0)
 		for _, ingress := range il.Items {
 			ingressClass := getAnnotationsString(ingress.Metadata.Annotations, ingressClassAnnotation, "")
 			for _, v := range a.ingressFilters {
 				if v == ingressClass {
-					ret = append(ret, a.newIngressFromKube(ingress))
+					ing := a.newIngressFromKube(ingress)
+					if ing != nil {
```

Comment:
This nil check is no longer needed as we don't return `nil` anymore

Reply:
good catch, much cleaner to read :)